setup.py

```diff
@@ -1,114 +1,53 @@
     ("etc/jupyter/jupyter_notebook_config.d", "etc/jupyter/jupyter_notebook_config.d", "voila-gridstack.json"),
     ("etc/jupyter/nbconfig/notebook.d", "etc/jupyter/nbconfig/notebook.d", "voila-gridstack.json"),
     ("share/jupyter/nbextensions/voila-gridstack", "voila-gridstack/static", "**"),
+    ("share/jupyter/labextensions/%s" % labext_name, str(lab_extension_dest), "**"),
+    ("share/jupyter/labextensions/%s" % labext_name, HERE, "install.json"),
     ("share/jupyter/nbconvert/templates/gridstack", "share/jupyter/nbconvert/templates/gridstack", "**")
 ]
 
-
-cmdclass = create_cmdclass("jsdeps",
-    package_data_spec=package_data_spec,
-    data_files_spec=data_files_spec
-)
-
-js_command = combine_commands(
-    install_npm(lab_extension_source, build_cmd="build:prod", npm=["jlpm"]),
-    ensure_targets(jstargets),
-)
-
-is_repo = os.path.exists(os.path.join(HERE, ".git"))
-if is_repo:
-    cmdclass["jsdeps"] = js_command
-else:
-    cmdclass["jsdeps"] = skip_if_exists(jstargets, js_command)
-
-base_develop_cmd = cmdclass['develop']
-
-
-class DevelopCmd(base_develop_cmd):
-    prefix_targets = [
-        ("nbconvert/templates", 'gridstack'),
-        ("voila/templates", 'gridstack')
-    ]
-    def run(self):
-        target_dir = os.path.join(sys.prefix, 'share', 'jupyter')
-        if '--user' in sys.prefix:  # TODO: is there a better way to find out?
-            target_dir = jupyter_core_paths.user_dir()
-        target_dir = os.path.join(target_dir)
-
-        for prefix_target, name in self.prefix_targets:
-            source = os.path.join('share', 'jupyter', prefix_target, name)
-            target = os.path.join(target_dir, prefix_target, name)
-            target_subdir = os.path.dirname(target)
-            if not os.path.exists(target_subdir):
-                os.makedirs(target_subdir)
-            rel_source = os.path.relpath(os.path.abspath(source), os.path.abspath(target_subdir))
-            try:
-                os.remove(target)
-            except:
-                pass
-            print(rel_source, '->', target)
-            os.symlink(rel_source, target)
-
-        super(DevelopCmd, self).run()
-
-
-cmdclass['develop'] = DevelopCmd if jupyter_core_paths else base_develop_cmd
-
-with open("README.md", "r") as fh:
-    long_description = fh.read()
+# Get the package info from package.json
+pkg_json = json.loads((HERE / "packages/jupyterlab-gridstack/package.json").read_bytes())
+long_description = (HERE / "README.md").read_text()
 
 setup_args = dict(
-    name=name,
-    version="0.2.0",
-    url="https://github.com/voila-dashboards/voila-gridstack",
-    author="Voila Development Team",
-    author_email="jupyter@googlegroups.com",
-    description="A GridStack template for Voila.",
+    name=NAME,
+    version=pkg_json["version"],
+    url=pkg_json["homepage"],
+    author=pkg_json["author"],
+    description=pkg_json["description"],
```

Comment:
Maybe we still want to keep the metadata for the Python package separate from the JupyterLab extension?

So we don't use this description for the template:

https://github.com/voila-dashboards/voila-gridstack/blob/99c1b35723d41699f7d90a702d795b042e8f8e8e/packages/jupyterlab-gridstack/package.json#L4

Reply:
That's right, I'll change. I'll use the `setup.cfg` to add the metadata for the python package then.

Reply:
And we can even move most of this to `setup.cfg`, similar to: https://github.com/voila-dashboards/voila/blob/main/setup.cfg